### PR TITLE
Dev build

### PR DESCRIPTION
**Pull request by Wovchena**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Required for https://github.com/openvinotoolkit/openvino.genai/pull/543

### CODE REVIEW

**Comment by Wovchena** [comment]
@conda-forge-admin, please rerender

**Comment by Wovchena** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
 package:
   name: {{ name|lower }}-split
   version: {{ version }}
 
 source:
-  - url: https://github.com/openvinotoolkit/openvino/archive/refs/tags/{{ version }}.tar.gz
-    sha256: a972ed6525889f63e0de1f2484dd8e4c5ed649a64da454caf36986bc40c80a44
+  - url: https://github.com/Wovchena/openvino/archive/4235e7523052ba34a5bd7135b9fd419643021756.tar.gz
+    sha256: b5cb38553d4bc848823577ec6a95a4ec959a92202241da41f007e42a1be20db4
```
Comment (ilya-lavrenov) [inline recipe/meta.yaml:11]: Why non official repo is used?

Comment (Wovchena) [reply recipe/meta.yaml:11]: It doesn't work: https://github.com/conda-forge/openvino-feedstock/pull/100/commits/2831159dd5124b18f07dd39fa22ba54b82f0d628

Comment (ilya-lavrenov) [reply recipe/meta.yaml]: ok, let's fix in original place first

**Pull request merged**